tests/components/bond/test_cover.py

```diff
@@ -19,18 +18,15 @@
 
 _LOGGER = logging.getLogger(__name__)
 
+TEST_DEVICE_IDS = ["device-1"]
+TEST_DEVICE = {"name": "name-1", "type": BOND_DEVICE_TYPE_MOTORIZED_SHADES}
+
 
 async def test_entity_registry(hass):
     """Tests that the devices are registered in the entity registry."""
 
-    with patch(
-        "homeassistant.components.bond.utils.get_bond_devices",
-        return_value=[
-            BondDevice(
-                "device-1",
-                {"name": "name-1", "type": BOND_DEVICE_TYPE_MOTORIZED_SHADES},
-            )
-        ],
+    with patch("bond.Bond.getDeviceIds", return_value=TEST_DEVICE_IDS), patch(
```

Comment:
Is this patch target path safe? Are we patching in the correct place?

Reply:
i guess it could be scoped to the import in component. was following other paths

Reply:
I would use `"homeassistant.components.bond.Bond.getDeviceIds"`. That should work regardless of how the library is structured.